react/index.js

```diff
@@ -100,7 +100,7 @@ class ProductSummary extends Component {
     return path(['sku', 'seller', 'commertialOffer'], this.props.product)
   }
 
-  renderImage = () => {
+  get renderImage() {
```

Comment:
why use the `get`? this is more like a method than a computed property, and makes more sense to call it as a ordinary method.

Reply:
I think that if the get will be used would be better to remove the 'render' prefix. I think the get could be used since these methods dont change any fields

Reply:
`get` is fine, as @brunojdo said

Reply:
The idea of using getters is because is lazy. This fact wouldn't make it better use getters instead of ordinary method? @lucasecdb

Reply:
agree with @rerissondaniel